src/styles/utils/variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
```

Comment:
its better to give more clear name like $main-color, $secondary-color
 
also please write colors in lowercase like #060b35; #00acdc;

Reply:
do it!